src/main.cpp

```diff
@@ -669,9 +669,15 @@ Args parse_args(int argc, char* argv[])
     if (optind == argc) {
       args.search = "*:*";
     } else if (optind == argc - 1) {
-      args.search = argv[optind];
-      if (args.search == "*") {
+      std::string val(argv[optind]);
```

Comment:
```suggestion
      std::string_view val(argv[optind]);
```
Just to eliminate a string copy, but string copies are a big cause of overhead in bpftrace at the moment so every little helps.

Also applies to the new helper functions in utils.h